Remotion/Web/Development.WebTesting.IntegrationTests/MultiWindowTest.cs

```diff
@@ -27,6 +28,21 @@ namespace Remotion.Web.Development.WebTesting.IntegrationTests
   [TestFixture]
   public class MultiWindowTest : IntegrationTest
   {
+    [Test]
+    public void TestMultiWindowResizeBrowserContent ()
+    {
+      const string windowTitle = "MyWindow";
+      var home = Start();
+      var newSize = new Size (600, 600);
+      var loadWindowFunctionInNewWindowButton = home.WebButtons().GetByID ("LoadWindowFunctionInNewWindow");
+      var window1 = loadWindowFunctionInNewWindowButton.Click().ExpectNewPopupWindow<WxePageObject> (windowTitle);
+      var window2 = loadWindowFunctionInNewWindowButton.Click().ExpectNewPopupWindow<WxePageObject> (windowTitle);
+
+      Helper.BrowserConfiguration.BrowserHelper.ResizeBrowserContentTo (window2.Context.Window, newSize);
+
+      Assert.That (Helper.BrowserConfiguration.BrowserHelper.GetBrowserContentBounds (window1.Context.Window).Size, Is.EqualTo (newSize));

```

Comment:
Should we have a second test that tests that window 2 can also be selected and it not just happens to always select the first window?